### ISSUE TITLE
Age restricted tracks are broken

### ISSUE THREAD

**Issue by aikaterna** [Walkyst/lavaplayer-fork#17]
Age restricted tracks are broken again. This is a Lavalink trace but I am using your lavaplayer @ 1.3.96.

Tracks tried on US East: B3eAMGXFw1o, GV7FSJDwjME, jPFfDavVpj0, i63cgUeSsY0

```
2021-12-18 10:23:32.416  INFO 1045 --- [lava-daemon-pool-info-loader-2-thread-1] lavalink.server.player.AudioLoader       : Loaded track LITTLE BIG - BIG DICK
2021-12-18 10:23:32.452  INFO 1045 --- [XNIO-1 I/O-1] lavalink.server.io.SocketServer          : {"op": "play", "guildId": "REDACTED", "track": "QAAAeQIAFUxJVFRMRSBCSUcgLSBCSUcgRElDSwAKTGl0dGxlIEJpZwAAAAAAAquYAAtpNjNjZ1VlU3NZMAABACtodHRwczovL3d3dy55b3V0dWJlLmNvbS93YXRjaD92PWk2M2NnVWVTc1kwAAd5b3V0dWJlAAAAAAAAAAA=", "noReplace": false, "startTime": "0", "pause": false}
2021-12-18 10:23:32.528  WARN 1045 --- [lava-daemon-pool-playback-1-thread-1] c.s.d.l.s.y.DefaultYoutubeTrackDetails   : Video i63cgUeSsY0 with no detected format field, response {"responseContext":{"visitorData":"CgtiMDdFZ1hSNlVRdyikzviNBg%3D%3D","serviceTrackingParams":[{"service":"GFEEDBACK","params":[{"key":"is_viewed_live","value":"False"},{"key":"logged_in","value":"0"},{"key":"e","value":"24142941,24115641,24141079,24036947,24002923,24136746,24007790,24135236,24143411,24140247,23983296,24138442,23884386,23966208,24021124,24143428,23946420,24077241,24142961,1714254,24004644,24059148,24007246,24142596,23944779,24058380,23744176,24091230,24054345,24001373,24135228,24132215,24080738,24063124,24143789,24116735,24117476,24128155,23888717,23918597,24060091,23735348,24082661,24131027,24085811,39321475,23934970,24109690,24104858,24116717,24034168,24134152,23882502,24108823,24149573,24110902,24106839,23998056,24095695,24054937,24077266,24028143,24123943,23968386,24130239"}]},{"service":"CSI","params":[{"key":"c","value":"ANDROID"},{"key":"cver","value":"16.24"},{"key":"yt_li","value":"0"},{"key":"GetPlayer_rid","value":"0xea78bc0b682772ef"}]},{"service":"GUIDED_HELP","params":[{"key":"logged_in","value":"0"}]},{"service":"ECATCHER","params":[{"key":"client.version","value":"16.24"},{"key":"client.name","value":"ANDROID"},{"key":"client.fexp","value":"24142941,24115641,24141079,24036947,24002923,24136746,24007790,24135236,24143411,24140247,23983296,24138442,23884386,23966208,24021124,24143428,23946420,24077241,24142961,1714254,24004644,24059148,24007246,24142596,23944779,24058380,23744176,24091230,24054345,24001373,24135228,24132215,24080738,24063124,24143789,24116735,24117476,24128155,23888717,23918597,24060091,23735348,24082661,24131027,24085811,39321475,23934970,24109690,24104858,24116717,24034168,24134152,23882502,24108823,24149573,24110902,24106839,23998056,24095695,24054937,24077266,24028143,24123943,23968386,24130239"}]},{"service":"LISTNR","params":[{"key":"e","value":"24085324,24106839,24033252,24110902,24030282,24109931,24024517,24116717,24116735,24117476,24141079,39321475,24115641,24036947,24140247,24142941,24104858,24132215,24058380,24109690,24054345,24059148,24130239,24134152,24143428,24037284"}]},{"service":"SUGGEST","params":[{"key":"sugexp","value":"shwpc1,ytpo.bo.me=0,ytposo.bo.me=0,ytpo.bo.ei=45364322,ytposo.bo.ei=45364322,ytpo.bo.uo.et=1,ytposo.bo.uo.et=1,ytpo.bo.uo.mw=10,ytposo.bo.uo.mw=10,ytpo.bo.uo.md=7,ytposo.bo.uo.md=3,cfro=1,ytpo.bo.me=1,ytposo.bo.me=1,ytpo.bo.ei=45351862,ytposo.bo.ei=45351862,ytpo.bo.uo.et=1,ytposo.bo.uo.et=1,ytpo.bo.uo.mw=10,ytposo.bo.uo.mw=10,ytpo.bo.uo.md=7,ytposo.bo.uo.md=3"}]}]},"trackingParams":"CAAQu2kiEwiRxZKG_O30AhXhS0wIHXp8CsY=","playabilityStatus":{"status":"LOGIN_REQUIRED","reason":"This video may be inappropriate for some users.","errorScreen":{"confirmDialogRenderer":{"title":{"runs":[{"text":"You must sign in to view this video"}]},"trackingParams":"CAIQxjgiEwiRxZKG_O30AhXhS0wIHXp8CsY=","dialogMessages":[{"runs":[{"text":"This video may be inappropriate for some users."}]}],"confirmButton":{"buttonRenderer":{"style":"STYLE_BLUE_TEXT","size":"SIZE_DEFAULT","isDisabled":false,"text":{"runs":[{"text":"Sign in"}]},"trackingParams":"CAQQ8FsiEwiRxZKG_O30AhXhS0wIHXp8CsY="}},"cancelButton":{"buttonRenderer":{"style":"STYLE_BLUE_TEXT","size":"SIZE_DEFAULT","isDisabled":false,"text":{"runs":[{"text":"Go back"}]},"trackingParams":"CAMQ8FsiEwiRxZKG_O30AhXhS0wIHXp8CsY="}}}},"desktopLegacyAgeGateReason":1,"reasonTitle":"You must sign in to view this video","contextParams":"Q0FFU0FnZ0I="},"videoDetails":{"videoId":"i63cgUeSsY0","title":"LITTLE BIG - BIG DICK","lengthSeconds":"175","keywords":["LITTLE BIG","BIG","dick","music","rave","ilich","klick","klak","olympiya","gokk","pasok","russia","die antwoord","sex","body","cute","dwarf","prusikin","boobs","ass","bodyart","color","fashion","finger","dance","dance music","electronic","big","little big","little","littlebig","watch little big shots"],"channelId":"UCu7TZ_ATWgjgD9IrNLdnYDA","isOwnerViewing":false,"shortDescription":"Очки и Ничоси отсюда: http://maskbro.ru \nSubscribe: http://bit.ly/2gdHQs8 \nNew album \"FUNERAL RAVE\" available on iTunes now: http://apple.co/2hKUQq0\n\nOfficial merch / Официальный мерч:\nhttps://bit.ly/2WAmMBN\n\nMail to: thelittlebigrussia@gmail.com \n\nBooking (Ru, USA):  littlebig.booking@gmail.com\n+79816863508\n\nBooking (Europe): +33 608 815 807 Fabristi\nhttp://www.base-productions.com\n\nLITTLE BIG:\nNew album \"FUNERAL RAVE\" available on iTunes now: https://itunes.apple.com/album/id1068508618?ls=1&app=itunes\nfacebook: https://www.facebook.com/littlebigbandofficial/\nsoundcloud: https://soundcloud.com/littlebigrussia\nvk: http://vk.com/littlebigband\ntwitter: https://twitter.com/LITTLEBIG_BAND\nwebsite: http://thelittlebig.com\n\nLITTLE BIG production:\n\nDirected:  Alina Pasok, Ilya Prusikin\nDOP: Yriy Iskhakov\nCamera Assistants and focus pullers: \nFedor Yashkin\nGaffer: Maksim Kolupaev\nColor: Alina Pasok, Aleksandr Karelin\n\nProducer: Alina Pasok, Denis Glazin\nLine Producer: Sergey Makarov\nArt Director: Daria Uhvatova \nEditing: Alina Pasok, Ilya Prusikin\nVFX: Aleksandr Karelin\nStyle: Vadim Ksendokhov\nDancers:  Alyona Bonchinche, Katya Bonchinche, Dima Bonchinche, Timofey Smirnov\nMUA: Maria Francevich \nBodyart: Ksenya Eliseeva vk.com/eliseeva_lab\nHairstyle: Marina Koskova\nAdministrators: Vasilisa Lubenko\nBodies and Lips:  Alyona Dvoichenkova, Irina Smelaya, Vasilisa Lubenko, Polina Soshnikova\n\nCrew and Runners: Ivan Romanov, Denis Glazin, Irina Smelaya, Vadim Tolmachev, Masha Lesovaya, Mikhail Kshishtovsky, Aleksander Anokhin, Jenya Nikolaev\n\nmake up: Elena Pentenshina, Maria Francevich \nMusic: Sergey Makarov, Iliya Prusikin (Mixing: Aleksander Karelin)\n\nLyrics: Sergey Rahmaninov, Ilya Prusikin, Sergey Makarov\n\nLittle Big band: Iliya Prusikin, Sergey Gokk Makarov, Olympiya Ivleva and Mr. Clown (Anton Lissov) \n\nSpecial Thanks: Sofa Taurskaya\nEldar Djarakhov\nGeorgy Makelroy\nAnastasia Kharlamova\nMr. BrownGreen\nПарикмахерской «Гнездо» https://vk.com/gnezdosalon\nАгентству \"Сметана\" http://vk.com/smetanaagency","isCrawlable":true,"thumbnail":{"thumbnails":[{"url":"https://i.ytimg.com/vi/i63cgUeSsY0/default.jpg","width":120,"height":90},{"url":"https://i.ytimg.com/vi/i63cgUeSsY0/mqdefault.jpg","width":320,"height":180},{"url":"https://i.ytimg.com/vi/i63cgUeSsY0/hqdefault.jpg","width":480,"height":360},{"url":"https://i.ytimg.com/vi/i63cgUeSsY0/sddefault.jpg","width":640,"height":480}]},"allowRatings":true,"viewCount":"82310706","author":"Little Big","isPrivate":false,"isUnpluggedCorpus":false,"isLiveContent":false},"playerSettingsMenuData":{"loggingDirectives":{"trackingParams":"CAEQtc4GIhMIkcWShvzt9AIV4UtMCB16fArG","visibility":{"types":"12"}}},"frameworkUpdates":{"entityBatchUpdate":{"mutations":[{"entityKey":"Eg0KC2k2M2NnVWVTc1kwIPYBKAE%3D","type":"ENTITY_MUTATION_TYPE_REPLACE","payload":{"offlineabilityEntity":{"key":"Eg0KC2k2M2NnVWVTc1kwIPYBKAE%3D","accessState":"OFFLINEABILITY_FEATURE_ACCESS_STATE_UNKNOWN","addToOfflineButtonState":"ADD_TO_OFFLINE_BUTTON_STATE_UNKNOWN"}}}],"timestamp":{"seconds":"1639851812","nanos":520389018}}}} polymer null
2021-12-18 10:23:32.535  WARN 1045 --- [lava-daemon-pool-playback-1-thread-1] c.s.d.l.t.p.LocalAudioTrackExecutor      : Suspicious exception for playback of i63cgUeSsY0

com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Unable to play this YouTube track.
	at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeTrackDetails.loadTrackFormats(DefaultYoutubeTrackDetails.java:79) ~[lavaplayer-fork-1.3.96.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeTrackDetails.getFormats(DefaultYoutubeTrackDetails.java:49) ~[lavaplayer-fork-1.3.96.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioTrack.loadBestFormatWithUrl(YoutubeAudioTrack.java:83) ~[lavaplayer-fork-1.3.96.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioTrack.process(YoutubeAudioTrack.java:42) ~[lavaplayer-fork-1.3.96.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:104) ~[lavaplayer-fork-1.3.96.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$1(DefaultAudioPlayerManager.java:348) ~[lavaplayer-fork-1.3.96.jar!/:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:834) ~[na:na]
	Suppressed: com.sedmelluq.discord.lavaplayer.tools.exception.EnvironmentInformation: 
  lavaplayer.version: 1.3.96-original
  os.arch: x86_64
  os.name: Mac OS X
  os.version: 10.15.7
  java.vendor: AdoptOpenJDK
  java.version: 11.0.9
  java.runtime.version: 11.0.9+11
  java.vm.version: 11.0.9+11
Caused by: java.lang.IllegalStateException: No track formats found.
	... 9 common frames omitted
```

**Comment by Walkyst** [comment]
Currently default YT account used in my fork is disabled by YT, so all age restricted videos are not working, because of that I don't see any point to hardcode new account, instead in GitHub repo will be guide soon how setup own account for avoiding restrictions

**Comment by Walkyst** [comment]
Released 1.3.97 and https://github.com/Walkyst/lavaplayer-fork/issues/18, so this should help with age restricted videos
